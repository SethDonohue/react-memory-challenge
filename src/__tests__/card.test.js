import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../card/card';

test('renders a card', () => {
  const component = renderer.create(
    <Card 
      key={0}
      id={0}
      bgColor="gray"
      value={10}
      color="rgb(0, 100, 0)"
      display="block"
      matched={false}
      flipped={false}
      matchCheck="match check function"
    />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
