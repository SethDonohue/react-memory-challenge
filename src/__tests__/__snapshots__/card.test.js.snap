// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a card 1`] = `
<div
  className="card"
  id={undefined}
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "gray",
    }
  }
  type="button"
>
  <p
    className="card-text"
    style={
      Object {
        "display": "block",
      }
    }
  >
    10
  </p>
</div>
`;
